tests/run-make/codegen-options-parsing/Makefile

```diff
@@ -1,3 +1,5 @@
+# This test intentionally feeds invalid inputs to codegen and checks if the error message outputs contain the expected strings.
```

Comment:
Is this more like checking if codegen options parsing gives useful error messages upon encountering invalid options?